Automattic/jetpack
# Voice to Content: Fix upload file selection on iOS

Fixes #36164.

## Proposed changes:

* Change the `accept` attribute to list all allowed extensions instead of using the media type regex

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Run this branch on a JN site so it's easy to test on mobile
* Make sure to enable beta blocks on the JN site
* Go to the block editor, insert a Voice to Content block and upload an audio by clicking on "Upload audio"
* Confirm only audio files can be picked by default
   * This is were iOS was failing before, it would not allow selecting audio files
* Confirm the selected audio file will be processed correctly and the transcription will show up on the editor
* To avoid regressions, test it on:
   * Desktop with multiple OS and browser
   * Mobile with multiple OS and browser
   * It should work correctly on all platforms
   * The issue on iOS should be solved
